ModuleChangelog.md

```
@@ -1,3 +1,16 @@
+## 2.16.0
+Release Date: July 6, 2026
+
+#### RELEASE NOTES
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)